### PR TITLE
Set JAVA_VENDOR and JAVA_VENDOR_URL defines

### PR DESCRIPTION
**Pull request by pshipton**
Set OpenJ9 defaults for COMPANY_NAME, VENDOR_URL, VENDOR_URL_BUG, and
VENDOR_URL_VM_BUG, overriding the settings in
make/autoconf/version-numbers.

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

File: closed/autoconf/version-numbers [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2020, 2020 All Rights Reserved
+# ===========================================================================
+#
```
Comment (keithc-ca) [inline closed/autoconf/version-numbers:4]: Please remove the trailing whitespace (or the whole line); same goes for line 20.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: closed/autoconf/version-numbers (hunk 1 above)
Comment (pshipton) [reply closed/autoconf/version-numbers]: Fixed

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile osx jdk11 depends eclipse/openj9#11216

**Comment by keithc-ca** [comment]
Jenkins compile osx jdk11 depends eclipse/openj9#11216

**Pull request merged**